src/hotspot/share/cds/heapShared.cpp

```diff
@@ -280,10 +280,10 @@ oop HeapShared::archive_object(oop obj) {
     return ao;
   }
 
-  int len = obj->size();
+  int len = (int)obj->size();
```

Comment:
`obj->size()` already returns an `int` so I don't think the typecast is needed.
I agree with Ioi that this file shouldn't be changed.

Reply:
oop.hpp:105
  // Returns the actual oop size of the object
  inline size_t size();

Reply:
I will revert the change since it is not related to custom loader.